runtime/layers/function/bootstrap.php

```diff
@@ -17,16 +18,12 @@
 
 $lambdaRuntime = LambdaRuntime::fromEnvironmentVariable();
 
-$handlerFile = $appRoot . '/' . getenv('_HANDLER');
-if (! is_file($handlerFile)) {
-    $lambdaRuntime->failInitialization("Handler `$handlerFile` doesn't exist");
-}
-
-/** @noinspection PhpIncludeInspection */
-$handler = require $handlerFile;
+$container = Bref::getContainer();
 
-if (! $handler) {
-    $lambdaRuntime->failInitialization("Handler `$handlerFile` must return a function or object handler. See https://bref.sh/docs/runtimes/function.html");
+try {
+    $handler = $container->get(getenv('_HANDLER'));
+} catch (Throwable $e) {
+    $lambdaRuntime->failInitialization($e->getMessage());
```

Comment:
It looks like this replaces existing functionality; is it possible to still support `index.php` notation so that this ins't a BC break?

Reply:
Ignore me I now saw the FileHandlerLocator; tricksy.

Reply:
Yep! Total backward compatibility, and I don't even like calling this "backward" because using file handlers is still definitely a good thing, not deprecated at all.